steps/reasons-for-appealing/reason-for-appealing/template.html

```diff
@@ -57,8 +57,13 @@
 {% endblock %}
 
 {% block listItems %}
-    <p class="govuk-body">{{ content.dwpExplained | safe }}</p>
-    <p class="govuk-body">{{ content.readYouMRN | safe }}</p>
+    {% if isIba %}
+        <p class="govuk-body">{{ content.ibaExplained | safe }}</p>
```

Comment:
I think you can remove the " | safe" on these 4 - from my knowledge, they should only really be used if the content i.e. content.ibaExplained is HTML rather than just a string, so if in the content.en.json, if it was something like
"ibaExplained": "<p>some <strong>text</strong></p>" then | safe would be needed.

Reply:
This makes sense. I have removed them